docs/source/mayavi/conf.py

```diff
@@ -23,7 +23,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'traitsdoc']
+extensions = ['sphinx.ext.autodoc', 'sphinxext.traitsdoc']
```

Comment:
Is this needed?  The first line adds the sphinxext directory to the path, so shouldn't this just work?